src/jit/codegencommon.cpp

```diff
@@ -1642,6 +1642,16 @@ void CodeGen::genAdjustStackLevel(BasicBlock* block)
 {
 #if !FEATURE_FIXED_OUT_ARGS
     // Check for inserted throw blocks and adjust genStackLevel.
+    CLANG_FORMAT_COMMENT_ANCHOR;
+
+#if defined(UNIX_X86_ABI)
+    if (isFramePointerUsed() && compiler->fgIsThrowHlpBlk(block))
+    {
+        int offset = compiler->compCalleeRegsPushed * REGSIZE_BYTES + compiler->compLclFrameSize;
+        getEmitter()->emitIns_R_AR(INS_lea, EA_PTRSIZE, REG_SPBASE, REG_FPBASE, -offset);
+        SetStackLevel(0);
+    }
```

Comment:
This isn't necessary; the stack level is already set to zero on entry to this function.